src/engine/math_util.c

```diff
@@ -1264,7 +1282,7 @@ s32 anim_spline_poll(Vec3f result) {
 
 /**
  * @brief Checks if a ray intersects a surface using Möller–Trumbore intersection algorithm.
- * 
+ *
```

Comment:
awesome format change

Reply:
I don't think I can fix these now since it's funny line ending memes